docs/FAQ/identity_management_FAQ.md

```diff
@@ -1,18 +1,18 @@
 ## Identity Management (Membership Service)
 &nbsp;
 ##### What is unique about the fabric's Membership Service module?
-The design and implementation of the Membership Service module encompass many of the latest advances in cryptography, which we believe make it stand apart from other alternatives.
+One of the things that makes the Membership Service module stand out from the pack is our implementation of the latest advances in cryptography.
```

Comment:
Maybe we just omit the "What is unique..." Q and A. I like the rephrasing, but would not want to put the onus on you to do the research. Maybe remove and create an issue that suggests we should add the rephrased question with a suitable answer.